pyfolio/timeseries.py

```diff
@@ -72,57 +74,29 @@ def max_drawdown(returns):
     See https://en.wikipedia.org/wiki/Drawdown_(economics) for more details.
     """
 
-    if returns.size < 1:
-        return np.nan
-
-    df_cum_rets = cum_returns(returns, starting_value=100)
-    cum_max_return = df_cum_rets.cummax()
-
-    return df_cum_rets.sub(cum_max_return).div(cum_max_return).min()
+    return qrisk.max_drawdown(returns)
 
 
 def annual_return(returns, period=DAILY):
-    """Determines the annual returns of a strategy.
+    """Determines the mean annual growth rate of returns.
 
     Parameters
     ----------
     returns : pd.Series
         Periodic returns of the strategy, noncumulative.
-         - See full explanation in tears.create_full_tear_sheet.
+        - See full explanation in :func:`~pyfolio.timseries.cum_returns`.
     period : str, optional
-        - defines the periodicity of the 'returns' data for purposes of
-        annualizing. Can be 'monthly', 'weekly', or 'daily'
-        - defaults to 'daily'.
+        Defines the periodicity of the 'returns' data for purposes of
+        annualizing. Can be 'monthly', 'weekly', or 'daily'.
```

Comment:
I'll leave it in. The default is specified in the function definition so I removed it from the docstring.